### ISSUE TITLE
Change extensions 

### ISSUE THREAD

**Issue by hdolfen** [pyfsi/Kratos#15]
.msg extension is used by Abaqus for reporting. Therefore a different extension could be picked for CoCoNuT communication files. Evoking the remove_all_messages() method would also remove the Abaqus msg-files that can be useful for debugging.

Suggestions:
.cmsg
.coco

**Comment by toondm** [comment]
Good point, if you want to use the same message style for other solvers.
The only reason to add .msg is to be able to remove these files easily (e.g. during Finalize), without keeping track of which files were created. So yeah, we could call them .coco 😄 
As you can see [here](https://filext.com/list/c), .coco is still available!

**Comment by joris13** [comment]
.coco it is
